core/src/test/java/org/frankframework/filesystem/FileSystemTest.java

```diff
@@ -29,6 +30,30 @@ public void setUp() throws Exception {
 		autowireByName(fileSystem);
 	}
 
+	@Test
+	@Disabled //Works in master (8.1-SNAPSHOT), but not in lower versions, because of failures in AmazonS3FileSystemTest
```

Comment:
This extra information is best added to the `@Disabled("...")` annotation itself so that it can be logged by the test-runner for instance.

Reply:
Ah yes, thanks!